packages/wonder-blocks-accordion/src/components/accordion-section-header.tsx

```diff
@@ -20,6 +20,9 @@ type Props = {
     caretPosition: "start" | "end";
     // Corner roundedness type.
     cornerKind: AccordionCornerKindType;
+    // Whether the section is collapsible or not. If false, the header will
+    // not be clickable.
```

Comment:
**suggestion:** maybe it would be good to include that the contents are visible/expanded all the time?

Reply:
Ah, I didn't include that here since the content panel is not part of the header. I'll still add it now though.